packages/gafl-webapp-service/src/pages/licence-details/licence-start-date/route.js

```diff
@@ -1,4 +1,6 @@
-import { LICENCE_START_DATE, CONTROLLER } from '../../../constants.js'
+import { LICENCE_START_DATE, CONTROLLER } from '../../../uri.js'
+import { ADVANCED_PURCHASE_MAX_DAYS } from '@defra-fish/business-rules-lib/src/constants.js'
```

Comment:
This is importing from an explicit src file rather than the globally exported index file - see my other comment about exports in the business-rules-lib/index.js file